helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/constraints/SoftConstraintWeightModel.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.helix.controller.rebalancer.waged.constraints;
```

Comment:
Have you set up the apache license header in the intellij? Once you set it up, it should be automatically pumped up.